test/functional/test_runner.py

```diff
@@ -201,6 +201,7 @@
     'feature_backwards_compatibility.py --legacy-wallet',
     'feature_backwards_compatibility.py --descriptors',
     'wallet_txn_clone.py --mineblock',
+    'y2038.py',
```

Comment:
This naming convention doesn't work. Check out https://github.com/bitcoin/bitcoin/blob/a9335e4f129cadd87fda0fe49baf74670ded491a/test/functional/test_runner.py#L690
to see a list of accepted prefixes.

Reply:
Got it. Thanks. I've prefixed the name with example_. However, there is only one there and this tells me the prefix should be something else. 

I'm also wondering if this is the right approach to test this as the tests passes.